llvm/lib/IR/Globals.cpp

```diff
@@ -266,6 +266,16 @@ void GlobalValue::setNoSanitizeMetadata() {
   setSanitizerMetadata(Meta);
 }
 
+void GlobalValue::disableSanitizerMetadataGlobalTagging() {
+  if (!isTagged()) {
```

Comment:
the braces are not need for 1 liner condition. ( `git clang-format ` should handle it too)